### PR TITLE
ALPI-21

### PR DESCRIPTION
**Pull request by EduardoBeghe**
[ALPI-21](https://alpiusa.atlassian.net/secure/RapidBoard.jspa?rapidView=2&projectKey=ALPI&modal=detail&selectedIssue=ALPI-21)
- Added ISF Files validation and fields pre-fill, also added necessary components as pipes and datepickers
- Added some CSS style fixes

### CODE REVIEW

File: Web/AlpiUS.Web/src/app/processing/isf-process-result/isf-process-result.component.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export class IsfProcessResultComponent implements OnInit, OnChanges {
   public uncollapsedResult = false;
   public isfProcessResultForm: FormGroup;
   public invalidFields = [];
+  public ISFOffices: IsfPort[];
```
Comment (JuanBarrera-Rokolabs) [inline Web/AlpiUS.Web/src/app/processing/isf-process-result/isf-process-result.component.ts]: Should we make this camel case?

File: Web/AlpiUS.Web/src/app/processing/isf-process-result/isf-process-result.component.ts [hunk 2]
```diff
@@ -66,8 +67,18 @@ export class IsfProcessResultComponent implements OnInit, OnChanges {
...
+      { Name: 'Boston', Code: 'ALPUSABOS' },
+      { Name: 'Los Angeles', Code: 'ALPUSALAX' },
+      { Name: 'El Paso', Code: 'ALPUSAELP' },
+      { Name: 'Atlanta', Code: 'ALPUSAATL' },
+      { Name: 'Dallas', Code: 'ALPUSADFW' },
+      { Name: 'Chicago', Code: 'ALPUSAORD' },
+      { Name: 'Carolina', Code: 'ALPUSASJU' },
+      { Name: 'Burlington', Code: 'ALPUSAEWR' }
+    ]
     console.log(this.isfProcessResultForm);
```
Comment (JuanBarrera-Rokolabs) [inline Web/AlpiUS.Web/src/app/processing/isf-process-result/isf-process-result.component.ts]: Extra console.log

Comment (EduardoBeghe) [reply Web/AlpiUS.Web/src/app/processing/isf-process-result/isf-process-result.component.ts]: Fixed

File: Web/AlpiUS.Web/src/app/processing/isf-process-result/isf-process-result.component.ts (hunk 1 above)
Comment (EduardoBeghe) [reply Web/AlpiUS.Web/src/app/processing/isf-process-result/isf-process-result.component.ts]: Fixed

**Pull request merged**